lib/std/math/big/int_test.zig

```diff
@@ -2578,14 +2578,81 @@ test "big.int regression test for realloc with alias" {
 }
 
 test "big int popcount" {
-    var a = try Managed.initSet(testing.allocator, -1);
+    var a = try Managed.init(testing.allocator);
     defer a.deinit();
-    var b = try Managed.initSet(testing.allocator, -1);
-    defer b.deinit();
 
-    try a.popCount(&b, 16);
+    try a.set(0);
+    try popCountTest(&a, 0, 0);
+    try popCountTest(&a, 567, 0);
+
+    try a.set(1);
+    try popCountTest(&a, 1, 1);
+    try popCountTest(&a, 13, 1);
+    try popCountTest(&a, 432, 1);
+
+    try a.set(255);
+    try popCountTest(&a, 8, 8);
+    try a.set(-128);
+    try popCountTest(&a, 8, 1);
+
+    try a.set(-2);
+    try popCountTest(&a, 16, 15);
+    try popCountTest(&a, 15, 14);
+
+    try a.set(-2047);
+    try popCountTest(&a, 12, 2);
+    try popCountTest(&a, 24, 14);
+
+    try a.set(maxInt(u5000));
+    try popCountTest(&a, 5000, 5000);
+    try a.set(minInt(i5000));
+    try popCountTest(&a, 5000, 1);
+
+    // Check when values saturate all limbs, or just over/under.
+    const limb_bits = @bitSizeOf(Limb);
+    try a.set(-1);
+    try popCountTest(&a, 1, 1); // i1
+    try popCountTest(&a, 2, 2);
+    try popCountTest(&a, 16, 16);
+    try popCountTest(&a, 543, 543);
+    try popCountTest(&a, 544, 544);
+    try popCountTest(&a, limb_bits - 1, limb_bits - 1);
+    try popCountTest(&a, limb_bits, limb_bits);
+    try popCountTest(&a, limb_bits + 1, limb_bits + 1);
+    try popCountTest(&a, limb_bits * 2 - 1, limb_bits * 2 - 1);
+    try popCountTest(&a, limb_bits * 2, limb_bits * 2);
+    try popCountTest(&a, limb_bits * 2 + 1, limb_bits * 2 + 1);
+
+    try a.setString(16, "ff00000100000100" ++ ("0000000000000000" ** 62));
+    try popCountTest(&a, 4032, 10);
+    try popCountTest(&a, 6000, 10);
+    a.negate();
+    try popCountTest(&a, 4033, 48);
+    try popCountTest(&a, 4133, 148);
+
+    const limb_size = @bitSizeOf(Limb);
+    try a.setString(2, "1" ** limb_size);
+    try popCountTest(&a, 64, 64);
+    try popCountTest(&a, 65, 64);
+    try popCountTest(&a, 67, 64);
+    a.negate();
+    try popCountTest(&a, 126, 63);
+    try popCountTest(&a, 127, 64);
+    try popCountTest(&a, 128, 65);
+    try popCountTest(&a, 129, 66);
+    // TODO: Incorrect pop count for Mutable
```

Comment:
Please link the issue https://github.com/ziglang/zig/issues/13571 here.

Reply:
Thanks, I forgot to update this after I posted that issue.